docs/resources/index.rst

```diff
@@ -0,0 +1,147 @@
+.. _resources:
+
+=========
+Resources
+=========
+
+For the moment this folder hosts only one configuration file, ``config.yaml``.
+
+In the following we report an example:
+
+.. code-block:: yaml
+
+  # Example configuration file for PYIRF
+
+  # Based on protopipe one, it should progressively use GADF nomenclature.
+
+  general:
+    # part of the DL2 filename(s) common between particle types
+    template_input_file: '{}_onSource.S.3HB9-FD_ID0.eff-0-CUT0'
+     # Output table name
+    output_table_name: 'table_best_cutoff'
+     # where is the DL2 data
+    indir: '/Users/michele/Applications/ctasoft/tests/pyirf/EventDisplay/DL2/Paranal_20deg/CUT0'
+    # where to store DL3 data
+    outdir: '/Users/michele/Applications/ctasoft/tests/pyirf/EventDisplay/from_pyirf'
+
+  analysis:
+
+   # Theta square cut optimisation (opti, fixed, r68)
+   thsq_opt:
+    type: 'r68'
+    value: 0.2  # In degree, necessary for type fixed
+
+   # Normalisation between ON and OFF regions
+   alpha: 0.2
+
+   # Minimimal significance
+   min_sigma: 5
+
+   # Minimal number of gamma-ray-like
+   min_excess: 10
+
+   # Minimal fraction of background events for excess comparison
+   bkg_syst: 0.05
+
+   # Reco energy binning
+   ereco_binning:  # TeV
+    emin: 0.05
+    emax: 50
+    nbin: 21
+
+   # Reco energy binning
+   etrue_binning:  # TeV
+    emin: 0.05
+    emax: 50
+    nbin: 42
+
+  # =============================================================================
+  #                               TO REVIEW
+  # =============================================================================
+
+  particle_information:
+   gamma:
+    n_events_per_file: 22500000  #  10**5 * 10
+    n_files: 1
+    e_min: 0.05
+    e_max: 50
+    gen_radius: 1000
+    diff_cone: 0
+    gen_gamma: 2
+
+   proton:
+    n_events_per_file: 3750000000  #  2 * 10**5 * 20
+    n_files: 1
+    e_min: 0.01
+    e_max: 100
+    gen_radius: 2500
+    diff_cone: 1
+    gen_gamma: 2
+    offset_cut: 1.
+
+   electron:
+    n_events_per_file: 450000000  #  10**5 * 20
+    n_files: 1
+    e_min: 0.005
+    e_max: 5
+    gen_radius: 1000
+    diff_cone: 1
+    gen_gamma: 2
+    offset_cut: 1.
+
```

Comment:
I have to say I really don't like this config file.
But I will make a proposal in a separate PR.

Reply:
As with the rest, it is recycled material from protopipe, so we can change it later of course.
Yes, best to open a dedicated issue/PR for that.

Reply:
This goes in the same direction as other comments I made, but I actually think, we do not need a configuration system here.

We should aim for a  more library like use and expose everything via function / class parameters, not some configuration file or even system.

Reply:
Why not? there are a lot of things that are many configurable quantities

Reply:
Because that adds a burden for potential users to adapt the configuration system we choose here. We should just avoid that and provide functions that are nice to use.

scipy for example also as algorithms with many configuration options, but it does not force you to use "the scipy config system"

Reply:
perhaps it is not clear to me the difference between the two, we can certainly discuss this elsewhere

as long as the user can configure the optimization cuts and how IRFs production are produced it's fine

Reply:
The difference is basically if the functions expect some global `config` object like it is now, or just the few options relevant to that function.